src/js/component/RacingApp.js

```diff
@@ -0,0 +1,20 @@
+import { Racing } from "../domain/Racing.js";
+import { CarNamesForm } from "./CarNamesForm.js";
+import { CarTrackForm } from "./CarTrackForm.js";
+import { TryCountForm } from "./TryCountForm.js";
+
+export class RacingApp {
+    constructor() {
+        this.racing = new Racing();
+
+        this.carNamesForm = new CarNamesForm(this.racing, {
+            onLoadTryForm: () => this.tryCountForm.display(),
+        });
+        this.tryCountForm = new TryCountForm(this.racing, {
+            onLoadCarTrackForm: () => this.carTrackForm.initialize(),
+        });
+        this.carTrackForm = new CarTrackForm(this.racing);
+    }
+
+    onLoadTryForm() {}
+}
```

Comment:
(개인적인 의견이니 꼭 반영하지 않으셔도 좋습니다.)
이 클래스는 하는 일이 앱을 initialize하는 것 밖에 없는 것 같습니다. 그래서 index.js 상에서도 `new RacingApp`으로 호출만 될 뿐 이렇게 생성된 인스턴스가 크게 사용되지 않고 있습니다.
차라리 함수로 구현해보면 어떨까요?

```suggestion
const initializeRacingApp = () => {
        const racing = new Racing();
        const carTrackForm = new CarTrackForm(racing);
        const tryCountForm = new TryCountForm(racing, {
            onLoadCarTrackForm: () => carTrackForm.initialize(),
        });
        const carNamesForm = new CarNamesForm(racing, {
            onLoadTryForm: () => tryCountForm.display(),
        });
    }
}
```